server/lib/allowed-features.ts

```diff
@@ -14,17 +15,18 @@ const FeatureAllowedForTypes = {
     types.PROJECT,
   ],
   [FEATURE.RECURRING_CONTRIBUTIONS]: [types.USER, types.ORGANIZATION, types.COLLECTIVE, types.FUND],
+  [FEATURE.RECEIVE_HOST_APPLICATIONS]: HOST_TYPES,
+  [FEATURE.HOST_DASHBOARD]: HOST_TYPES,
   [FEATURE.EVENTS]: [types.ORGANIZATION, types.COLLECTIVE],
   [FEATURE.PROJECTS]: [types.FUND],
   [FEATURE.USE_EXPENSES]: [types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT],
-  [FEATURE.RECEIVE_EXPENSES]: [types.ORGANIZATION],
+  [FEATURE.RECEIVE_EXPENSES]: [types.ORGANIZATION, types.COLLECTIVE, types.EVENT, types.FUND, types.PROJECT],
```

Comment:
@sbinlondon was this limitation to organizations made on purpose?

Reply:
Yes, I remember there being a conversation about how USE_EXPENSES vs RECEIVE_EXPENSES and only host orgs can RECEIVE_EXPENSES in the sense that they are the only ones who can pay them out?

Reply:
I see, it's true that they are the only ones who can pay them out but all accounts with a budget (collective, event, etc.) can receive them. In fact, most of the time you will submit your expense to a collective (ie. Babel) rather than a fiscal host (ie. OSC).